presto-function-namespace-managers/src/main/java/com/facebook/presto/functionNamespace/mysql/MySqlFunctionNamespaceManagerConfig.java

```diff
@@ -50,15 +50,15 @@ public MySqlFunctionNamespaceManagerConfig setFunctionsTableName(String function
     }
 
     @NotNull
-    public String getEnumTypesTableName()
+    public String getUserDefinedTypesTableName()
     {
-        return enumTypesTableName;
+        return userDefinedTypesTableName;
     }
 
-    @Config("enum-types-table-name")
-    public MySqlFunctionNamespaceManagerConfig setEnumTypesTableName(String enumTypesTableName)
+    @Config("user-defined-types-table-name")
```

Comment:
@highker @rongrong  When renaming configuration options are we supposed to remove them or just deprecate them (and the older version still works?)? This ended up breaking 0.257 of Sapphire because the config there supplied the old name for this property.